.github/workflows/reusable_workflow.yaml

```diff
@@ -19,7 +19,13 @@ on:
         type: string
       github_namespace:
         required: false
-        type: string    
+        type: string
+      archetype:
+        required: false
+        type: string
+      multi_arch:
+        required: false
+        type: boolean   
```

Comment:
Why do you have multi_arch in some spots, and req_check_only in others?

Reply:
I gave this a lot of thought. It depends on the context. In the workflow you would be invoking you want a multi-arch build, so ACT would need to internally not attempt to build and push, as that would be docker buildx. Thus a req_check would not make sense in context. While if you were running this locally, you would not be using it for multi-arch builds, only for a requirement check as ACT cannot specifically create multi-arch builds. But both use cases would need ACT to perform the same.

Reply:
Personally, I think req_check_only makes the most sense everywhere. Since no matter what all it does is disable the build and push. Then having a requirement check plus a separate build makes sense, as opposed to both a build and a multi-arch build.